pyuvdata/parameter.py

```diff
@@ -225,6 +238,25 @@ def __eq__(self, other):
                     print(f"{self.name} parameter value is array, values are not close")
                     return False
             else:
+                # check to see if strict types are used
+                if self.strict_type:
+                    # types must match
+                    if not isinstance(self.value, other.expected_type):
+                        print(
+                            f"{self.name} parameter has incompatible types. Left "
+                            f"requires {type(self.value)}, right is "
+                            "{other.expected_type}"
```

Comment:
looks like we're missing an `f` here?
```suggestion
                            f"{other.expected_type}"
```